.github/workflows/weekly-base-build.yaml

```diff
@@ -35,21 +35,69 @@ jobs:
           BUILD_DATE=$(TZ='US/Los_Angeles' date '+%Y-%m-%d')
           echo "BUILD_DATE=${BUILD_DATE}" >> $GITHUB_OUTPUT
 
-  build:
+  amd64:
     needs: metadata
     uses: ./.github/workflows/_build_base.yaml
     with:
+      ARCHITECTURE: amd64
+      BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
+    secrets: inherit
+
+  arm64:
+    needs: metadata
+    uses: ./.github/workflows/_build_base.yaml
+    with:
+      ARCHITECTURE: arm64
       BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
     secrets: inherit
 
   publish:
```

Comment:
Any reason this file doesn't use _publish_container.yaml?

Reply:
I think I might just forget to to so. Done in ce8ba2d.